docs/layers/h3-hexagon-layer.md

```diff
@@ -0,0 +1,131 @@
+
+### Render Options
+
+##### `coverage` (Number, optional)
+
+* Default: `1`
+
+Radius multiplier, between 0 - 1. The radius of the disk is calculated by
+`coverage * radius`
+
+##### `elevationScale` (Number, optional)
+
+* Default: `1`
+
+Column elevation multiplier. The elevation of column is calculated by
+`elevationScale * getElevation(d)`. `elevationScale` is a handy property
+to scale all hexagon elevations without updating the data.
+
+##### `extruded` (Boolean, optional)
+
+* Default: `true`
+
+Whether to extrude hexagon. If se to false, all hexagons will be set to flat.
+
+##### `fp64` (Boolean, optional)
+
+* Default: `false`
+
+Whether the layer should be rendered in high-precision 64-bit mode. Note that since deck.gl v6.1, the default 32-bit projection uses a hybrid mode that matches 64-bit precision with significantly better performance.
+
+##### `material` (Object, optional)
+
+* Default: `new PhongMaterial()`
+
+This is an object that contains material props for [lighting effect](/docs/effects/lighting-effect.md) applied on extruded polygons.
+
+
+### Data Accessors
+
+##### `getHexagon` (Function, optional)
+
+* Default: `object => object.hexagon`
+
+Method called to retrieve the [H3](https://uber.github.io/h3/) hexagon index of each object. Note that all hexagons within one `H3HexagonLayer` must use the same [resolution](https://uber.github.io/h3/#/documentation/core-library/resolution-table).
+
+##### `getColor` (Function|Array, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `[255, 0, 255, 255]`
+
+The rgba color of each object, in `r, g, b, [a]`. Each component is in the 0-255 range.
+
+* If an array is provided, it is used as the color for all objects.
+* If a function is provided, it is called on each object to retrieve its color.
+
+##### `getElevation` (Function|Number, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `1000`
+
+The elevation of each cell in meters.
+
+* If a number is provided, it is used as the elevation for all objects.
+* If a function is provided, it is called on each object to retrieve its elevation.
+
+
+## Sub Layers
+
+The H3HexagonLayer renders the following sublayers:
+
+* `hexagon-cell` - a [ColumnLayer](/docs/layers/column-layer.md) rendering all hexagons.
+
+
+## Remarks
+
+* Each hexagon in the H3 indexing system is [slightly different in shape](https://uber.github.io/h3/#/documentation/core-library/coordinate-systems). To draw a large number of hexagons efficiently, the H3HexagonLayer assumes that all hexagons within the current viewport have the same shape as the one at the center of the current viewport. This strategy is usually sufficient. However, the discrepancy may become visually significant at rare geolocations. In that case, the [H3ClusterLayer] can be used as an alternative by trading performance for accuracy.
```

Comment:
It is mentioned in the description of `getHexagon`.

Reply:
If I can interject, every aperture has a mix of hexagons and pentagons. The pentagons have simply been placed in water so common use-cases do not run into them.

Reply:
This will require some changes in the `ColumnLayer`, will do in a follow up PR. Do you have a hex id I can test with?

Reply:
Sure. One example pentagon at resolution 9 is `891c0000003ffff`.

Reply:
(To be fair, when I said "a mix of hexagons and pentagons" there are always 12 pentagons at each resolution, so the higher the resolution it becomes far, far less likely to run into one.)

You may also want to run a `kRing` of 10 or 20 around that pentagon, since pentagons are at the epicenter of the hexagon distortions and you're going to run into essentially 5 different hexagon shapes in that area (one for each of the icosahedron faces the hexagons are on, and technically some really weird ones that are split across the icosahedron boundaries).